src/components/Dashboard/widgets/Wallet/TxInformation.tsx

```diff
@@ -56,6 +56,7 @@ const TxInformation = ({
 }: TxConfirmationProps): ReactElement => {
   const tokenTx = useSelector(selectCurrenTxTokenTx);
   const currentToken = useSelector(selectCurrentTokenInfo);
+  const txAmount = toBitcoinAmount(amount);
   // const usdValueTemp = formatFiat(Number(amount) * Number(usdValue));
```

Comment:
clean up?

Reply:
I guess I could get rid of those, these are commented out coz we have usd value in the designs, but we used to have no USD value as we werent tracked anywhere. It doesnt look like anyone is missing it though. I ll remove it.